Update index.md

annajiat opened the pull request:
Hi,

Under the preparations section, the 2nd reading, "The Carpentries 2018 Annual Report" URL leads to an error. It may be fixed as follows:

Current 404 URL:
https://carpentries.org/files/assessment/TheCarpentries2018AnnualReport.pdf

Proposed new URL:
https://carpentries.org/files/reports/TheCarpentries2018AnnualReport.pdf

annajiat committed:
Update index.md

Hi,

Under the preparations section, the 2nd reading, "The Carpentries 2018 Annual Report" URL leads to an error. It may be fixed as follows:

Current 404 URL:
https://carpentries.org/files/assessment/TheCarpentries2018AnnualReport.pdf

Proposed new URL:
https://carpentries.org/files/reports/TheCarpentries2018AnnualReport.pdf

kariljordan commented:
Thank you SO MUCH! It would be great to link the 2019 Annual Report since that is available. @maneesha can you help?

maneesha commented:
Hi @annajiat Thanks for catching this! Can you update this  link to https://carpentries.org/files/reports/TheCarpentries2019AnnualReport.pdf so it has our most recent annual report?

annajiat committed:
Update index.md

Updated both the URL and the display text to 2019 Annual Report, Thank you @kariljordan , @maneesha .
https://carpentries.org/files/reports/TheCarpentries2019AnnualReport.pdf

annajiat commented:
Updated both the URL and the display text to 2019 Annual Report, Thank you @kariljordan , @maneesha .
https://carpentries.org/files/reports/TheCarpentries2019AnnualReport.pdf

maneesha commented:
@kariljordan  can you close this now?
